packages/daimo-api/src/landline/connector.ts

```diff
@@ -35,22 +35,46 @@ export function getLandlineURL(daimoAddress: string, sessionKey: string) {
 export async function getLandlineSession(
   daimoAddress: Address
 ): Promise<LandlineSessionKey> {
-  // @ts-ignore
-  const sessionKey = await landlineTrpc.getOrCreateSessionKey.mutate({
-    daimoAddress,
-  });
-  return sessionKey;
+  console.log(`[LANDLINE] getting session key for ${daimoAddress}`);
+
+  try {
+    // @ts-ignore
```

Comment:
why does this require a ts-ignore?

Reply:
the `landlineTrpc` is untyped rn since the trpc definition is in a separate repo. need to include ts-ignore otherwise the linter will say that the trpc method is unrecognized